README.md

```diff
@@ -121,6 +122,9 @@ Certain objects were changed after they were added to `typing`, and
   introspectable at runtime. In order to access overloads with
   `typing_extensions.get_overloads()`, you must use
   `@typing_extensions.overload`.
+- `NamedTuple` was changed in Python 3.11 to allow for multiple inheritance
+  with `typing.Generic` The implementation of `NamedTuple` was also changed in
+  3.9 so that `NamedTuple` became a function rather than a class.
```

Comment:
Why do we need to mention how it's implemented? How does it affect users?

Reply:
I included this because I thought it might be confusing for users on 3.8 if they realised that `typing.NamedTuple` was a class, but `typing_extensions.NamedTuple` was a function. (It _is_ a bit weird.) But I'm happy to delete it if you don't think that's an issue.

Reply:
I'd rather remove it, users shouldn't care about implementation details.